typescript/sdk/src/token/schemas.ts

```diff
@@ -0,0 +1,84 @@
+import { ethers } from 'ethers';
+import { z } from 'zod';
+
+import { routerConfigSchema } from '../router/schemas.js';
+
+import { TokenType } from './config.js';
+
+/// @notice This is a recreation of the ethersjs type, BigNumberish
+/// @dev consider moving this to a higher place if other schemas depend on this
+export const bigNumberIshSchema = z.union([
+  z.instanceof(ethers.BigNumber),
+  z.array(z.number()), // Bytes
+  z.bigint(),
+  z.string(),
+  z.number(),
+]);
```

Comment:
Why do we need this? We should use serialized version in configs like string. Keep in mind, these configs may be multi-protocol one day

Reply:
my guess is this was generated?

Reply:
Removed. will use `z.string().or(z.number())`